make -j8 install: man pages incorrectly don’t depend on jlibtool

In a clean checkout:

```
$ ./configure --prefix=/tmp/mfr
[…]

$ make -n /tmp/mfr/share/man/man8/rlm_redis_ippool_tool.8
INSTALL -d -m 755 /tmp/mfr/share/man/man8
make: build/make/jlibtool: Command not found
scripts/boiler.mk:656: recipe for target '/tmp/mfr/share/man/man8' failed
make: *** [/tmp/mfr/share/man/man8] Error 127

$ VERBOSE=1 make -n /tmp/mfr/share/man/man8/rlm_redis_ippool_tool.8             
mkdir -p build/tests/auth
rm -f build/tests/auth/depends.mk
for x in src/tests/auth/password_without_header src/tests/auth/md5_password src/tests/auth/wimax src/tests/auth/password_with_header src/tests/auth/user_password src/tests/auth/digest src/tests/auth/chap src/tests/auth/chap_header; do \
	y=`grep 'PRE: ' $x | sed 's/.*://;s/  / /g;s, , build/tests/auth/,g'`; \
	if [ "$y" != "" ]; then \
		z=`echo $x | sed 's,src/,build/',`; \
		echo "$z: $y" >> build/tests/auth/depends.mk; \
		echo "" >> build/tests/auth/depends.mk; \
	fi \
done
mkdir -p build/tests/keywords
echo INSTALL -d -m 755 /tmp/mfr/share/man/man8
build/make/jlibtool --silent --mode=install /tmp/upstream/4/install-sh -c -d -m 755 /tmp/mfr/share/man/man8
echo INSTALL rlm_redis_ippool_tool.8
build/make/jlibtool --silent --mode=install /tmp/upstream/4/install-sh -c -c -m 644 src/modules/rlm_redis_ippool/rlm_redis_ippool_tool.8  /tmp/mfr/share/man/man8/
```

It seems to me like build/make/jlibtool is not declared as a dependency, but should be. Is my assessment of the problem correct?

Taking a step back, I’m wondering why we install manpages using libtool in the first place. Shouldn’t using `install(1)` be sufficient?

--- a/scripts/install.mk
+++ b/scripts/install.mk
@@ -46,7 +46,7 @@ define ADD_INSTALL_RULE.exe
     install: $${${1}_INSTALLDIR}/$(notdir ${1})
 
     # Install executable ${1}
-    $${${1}_INSTALLDIR}/$(notdir ${1}): $${${1}_BUILD}/${1} | $${${1}_INSTALLDIR}
+    $${${1}_INSTALLDIR}/$(notdir ${1}): ${JLIBTOOL} $${${1}_BUILD}/${1} | $${${1}_INSTALLDIR}
 	@$(ECHO) INSTALL ${1}
 	$(Q)$${PROGRAM_INSTALL} -c -m 755 $${BUILD_DIR}/bin/${1} $${${1}_INSTALLDIR}/
 	$(Q)$${${1}_POSTINSTALL}
@@ -65,7 +65,7 @@ define ADD_INSTALL_RULE.a
     install: $${${1}_INSTALLDIR}/$(notdir ${1})
 
     # Install static library ${1}
-    $${${1}_INSTALLDIR}/$(notdir ${1}): ${1} | $${${1}_INSTALLDIR}
+    $${${1}_INSTALLDIR}/$(notdir ${1}): ${JLIBTOOL} ${1} | $${${1}_INSTALLDIR}
 	@$(ECHO) INSTALL ${1}
 	$(Q)$${PROGRAM_INSTALL} -c -m 755 $${BUILD_DIR}/lib/${1} $${${1}_INSTALLDIR}/
 	$(Q)$${${1}_POSTINSTALL}
@@ -87,7 +87,7 @@ define ADD_INSTALL_RULE.la
     install: $${${1}_INSTALLDIR}/$(notdir ${1})
 
     # Install libtool library ${1}
-    $${${1}_INSTALLDIR}/$(notdir ${1}): $${${1}_BUILD}/${1} | $${${1}_INSTALLDIR}
+    $${${1}_INSTALLDIR}/$(notdir ${1}): ${JLIBTOOL} $${${1}_BUILD}/${1} | $${${1}_INSTALLDIR}
 	@$(ECHO) INSTALL ${1}
 	$(Q)$${PROGRAM_INSTALL} -c -m 755 $${LOCAL_FLAGS_MIN} $${BUILD_DIR}/lib/${1} $${${1}_INSTALLDIR}/
 	$(Q)$${${1}_POSTINSTALL}
@@ -107,7 +107,7 @@ define ADD_INSTALL_RULE.man
     install: ${2}/$(notdir ${1})
 
     # Install manual page ${1}
-    ${2}/$(notdir ${1}): ${1} | ${2}
+    ${2}/$(notdir ${1}): ${JLIBTOOL} ${1} | ${2}
 	@$(ECHO) INSTALL $(notdir ${1})
 	$(Q)$${PROGRAM_INSTALL} -c -m 644 ${1} ${2}/
 
@@ -122,7 +122,7 @@ endef
 define ADD_INSTALL_RULE.dir
     # Install directory
     .PHONY: ${1}
-    ${1}:
+    ${1}: ${JLIBTOOL}
 	@$(ECHO) INSTALL -d -m 755 ${1}
 	$(Q)$${PROGRAM_INSTALL} -d -m 755 ${1}
 endef

